test/backend_binary_elementwise.in.cpp

```diff
@@ -160,14 +160,36 @@ NGRAPH_TEST(${BACKEND_NAME}, divide_int32)
 
     // Create some tensors for input/output
     auto a = backend->create_tensor(element::i32, shape);
-    copy_data(a, vector<int32_t>{0x40000140, 0x40000001, 8, 16});
+    copy_data(a, vector<int32_t>{-10, -10, 10, 10});
```

Comment:
Those unusual integer inputs where put there in order to check that the integer is not converted to a bfloat16. Please put those values back. There is not much point in having 4 of the same value to test divide.

Reply:
Let's have both. The values @silee2 put in are intended to test the new behavior.

Reply:
Restored modified test and added new test to test rounding mode.